packages/terra-base/src/Base.jsx

```diff
@@ -48,13 +48,21 @@ class Base extends React.Component {
 
   componentDidMount() {
     if (this.props.locale !== undefined) {
-      i18nLoader(this.props.locale, this.setState, this);
+      try {
+        i18nLoader(this.props.locale, this.setState, this);
```

Comment:
For React 16, it no longer renders the component if it errors out. https://reactjs.org/blog/2017/07/26/error-handling-in-react-16.html

We can use componentDidCatch, but it gets hard with this component, as we never defined what it should do in that case. We just let it error out previously, which no longer works for us as nothing will render. We can switch to the ErrorComponent when we have established what we would like to do when a locale isn't found.